### ISSUE TITLE
[Feature Request]: Provide ability to hide taskbar preview of window

### ISSUE THREAD

**Issue by pushkin-** [electron/electron#29033]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a feature request that matches the one I want to file, without success.

### Problem Description

I'd like to be able to hide the taskbar preview of a BrowserWindow when hovering over the taskbar icon.

### Proposed Solution

Add an API or configuration to `BrowserWindow` to allow turning off the taskbar preview in Windows.

### Alternatives Considered

Can be done in my .NET process that runs alongside the Electron app via [`DwmSetWindowAttribute`](https://docs.microsoft.com/en-us/windows/win32/api/dwmapi/nf-dwmapi-dwmsetwindowattribute) + the `DWMWA_FORCE_ICONIC_REPRESENTATION` flag

### Additional Information

n/a

**Comment by pushkin-** [comment]
Actually, what I really want I think is the ability to hide the preview when the window is minimized.

My app will "secure" itself after a certain amount of time, but if that happens while the window is minimized, Windows will still display the original preview image (pre securing) when hovering over the taskbar icon, and someone could potentially see some sensitive information. I want to hide that preview when minimized.

**Comment by miniak** [comment]
you can write a simple node module, which calls `DwmSetWindowAttribute`. You can get the HWND of the window by calling https://github.com/electron/electron/blob/main/docs/api/browser-window.md#wingetnativewindowhandle

**Comment by pushkin-** [comment]
@miniak Yeah I basically did this in my .NET child process, but was thinking Electron could handle it. Not sure if out of scope

**Comment by codebytere** [comment]
Closing in agreement with https://github.com/electron/electron/issues/29033#issuecomment-868778034 - this isn't something we're prepared to implement in the short term given it's easily addressable outside core.
